--- /home/chrismphilp/Documents/thirdYearProject/AutoInject/AutoInject/file_store/test/test1.py 18:32:24.712553 +0000
+++ /home/chrismphilp/Documents/thirdYearProject/AutoInject/AutoInject/file_store/test/patch_file.py 18:32:24.712614 +0000
@@ -1,11 +1,20 @@
+hello
+goodbye
 class adder:
-	def hello():
-		print("Say hello")
-	def goodbye(au_dieu):
-		return au_dieu
+    def goodbye():
+
+        print("Say goodbye")
+        print("hola")
+
+    def goodbye(au_revoir):
+        hello
+        y = au_revoir + 1
+        return y
+
+        return au_dieu
 
 class adder
-	def hello():
-		return "Hello"
+    def hello():
+        return "Hello"
 
 class adder:
